app/src/features/admin/users/AddSystemUsersForm.tsx

```diff
@@ -0,0 +1,165 @@
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Grid from '@material-ui/core/Grid';
+import IconButton from '@material-ui/core/IconButton';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import { mdiPlus, mdiTrashCanOutline } from '@mdi/js';
+import Icon from '@mdi/react';
+import CustomTextField from 'components/fields/CustomTextField';
+import { FieldArray, useFormikContext } from 'formik';
+import React from 'react';
+import yup from 'utils/YupSchema';
+
+export interface IAddSystemUsersFormArrayItem {
+  userIdentifier: string;
+  identitySource: string;
+  system_role: number;
+}
+
+export interface IAddSystemUsersForm {
+  participants: IAddSystemUsersFormArrayItem[];
```

Comment:
Maybe a bit of an annoying change, but can you change the `participants`/`participant` language to something like `users` or `systemUsers`.  "Participants" is something that only applies to Projects (in the database).  The code isn't wrong, but it will be slightly nicer to read for a future dev.